libs/sdk-embedding/src/iframe/kd.ts

```diff
@@ -1023,4 +1039,131 @@ export namespace EmbeddedKpiDashboard {
     ): obj is OpenScheduleEmailDialogCommandData {
         return getEventType(obj) === GdcKdCommandType.OpenScheduleEmailDialog;
     }
+
+    /**
+     * @public
+     */
+    export interface ISetFilterParentsAttributeFilter {
+        attributeFilter: {
+            displayForm: ObjRef;
+        };
+    }
+
+    /**
+     * @public
+     */
+    export type SetFilterParentsItemFilter = ISetFilterParentsAttributeFilter;
+
+    /**
+     * @public
+     */
+    export interface ISetFilterParentsItemParent {
+        parent: SetFilterParentsItemFilter;
+        connectingAttribute: ObjRef;
+    }
+
+    /**
+     * @public
+     */
+    export interface ISetFilterParentsItem {
+        filter: SetFilterParentsItemFilter;
+        parents: ISetFilterParentsItemParent[];
+    }
+
+    /**
+     * @public
+     */
+    export interface ISetFilterParentsDataBody {
+        filters: ISetFilterParentsItem[];
+    }
+
+    /**
+     * @public
+     */
+    export type SetFilterParentsCommandData = IGdcKdMessageEnvelope<
+        GdcKdCommandType.SetFilterParents,
+        ISetFilterParentsDataBody
+    >;
```

Comment:
I think you are missing the SetFilterParentCommand type def.

`SetFilterParentsCommand =  IGdcKdMessageEvent<..., ...>;`

Reply:
also, when you add the command type, add contractual documentation for the command. what it returns on success, what on error etc